### ISSUE TITLE
Keras GloVE embeddings example network is not learning. 

### ISSUE THREAD

**Issue by cjnolet** [fchollet/keras#3747]
I am getting the following when running the GloVE embedding example provided in the examples. I have made sure both Theano and Keras are up to date and I've copied the script ver-batim from your codebase (also based on the example from https://blog.keras.io/using-pre-trained-word-embeddings-in-a-keras-model.html).

Indexing word vectors.
Found 400000 word vectors.
Processing text dataset
Found 19997 texts.
Found 214909 unique tokens.
Shape of data tensor: (19997, 1000)
Shape of label tensor: (19997, 20)
Preparing embedding matrix.
Training model.
Train on 15998 samples, validate on 3999 samples
Epoch 1/50
15998/15998 [==============================] - 23s - loss: nan - acc: 0.0506 - val_loss: nan - val_acc: 0.0500
Epoch 2/50
15998/15998 [==============================] - 23s - loss: nan - acc: 0.0500 - val_loss: nan - val_acc: 0.0500

**Comment by dolaameng** [comment]
Hi @cjnolet : I got the same error when running the example. I suspect it was the 'relu' activation used in the Convlutional1D layer that causes this. Changing all of them to `linear` seems to fix this. May need more time to dig it...

**Comment by akhavr** [comment]
Works for me on latest version of keras.
